05LessonFive/LessonFive.js

```diff
@@ -16,5 +31,23 @@ module.exports = function() {
     // are replaced by the string "buzz", and the numbers divisble by both 3 and
     // 5 are replaced by fizzbuzz
 
+    var arr3 = [];
+
+    for (k = 0; k < 100; k++){
+      if (k % 3 === 0 && k % 5 === 0){
+        arr3.push('fizzbuzz');
+      }
+        else if (k % 3 === 0){
+          arr3.push('fizz');
+      }
+        else if (k % 5 === 0){
+          arr3.push('buzz')
+      }
+        else {
+          arr3.push(k);
+      }
+    }
+    return arr3;    
+
```

Comment:
These all look good! :+1: 

I want to make sure you know it's okay to have these arrays all named `arr` if you'd like. You don't have to worry about the names conflicting with the arrays inside other methods. This is because the variables declared inside of a method with `var` are local to that method - other methods cannot access that variable. So instead of `arr`, `arr2`, `arr3`, you could name them all `arr`.

If you already knew that, then disregard